spec/features/user_form_password_spec.rb

```diff
@@ -218,6 +218,13 @@ def fill_out_form(role: "Enumerator", email: true, admin: false)
           include_examples("generating new password")
           include_examples("entering new password")
         end
+
+        context "enumerator can see login instructions after he resets his password" do
+          let(:actor) { user }
+
+          include_examples("generating new password")
+          include_examples("entering new password with login instructions")
+        end
```

Comment:
Well done Jay! This is a great use of existing shared examples to test the thing you need to test. Did you confirm this failed before you made the change to make it pass?

Also, I'd like you to take this a step further. Note that the first example in this file is about the user editing their profile. So there is a bit of duplication here now. Can you find it? Can you clean it up? Happy to pair if you need.

Reply:
I am also now wondering if this would work for a staffer. We should probably have specs for all of them...